### PR TITLE
Bug/discussion close

### PR DESCRIPTION
**Pull request by ksowndar**
Bug is fixed in js file and post show page

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -64,32 +64,26 @@ def save
...
-    @saved_posts=Post.where(saved: true)
-  end
   def close
-  @post = Post.find(params[:id])
-  @post.closed
-  @post.update(closed: "true")
-end
+    @post = Post.find(params[:id])
+    @post.closed
+    @post.update(closed: "true")
```
Comment (iMagesh) [inline app/controllers/posts_controller.rb:77]: do we need these two lines?

File: app/javascript/includes/navbar.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
-$(document).ready(function(){
+$( document ).on('turbolinks:load', function() {
+ $(document).ready(function(){
```
Comment (iMagesh) [inline app/javascript/includes/navbar.js:2]: .ready is inside .on? that seems to be odd. what if the user reloads the page will this still work?

**Review by iMagesh — CHANGES_REQUESTED** [review]
I have added few comments. Please fix them and then we can merge.

File: app/javascript/includes/backtotop.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
-$(document).ready(function() {
+$( document ).on('turbolinks:load', function() {
+ $(document).ready(function() {
```
[the remark this answers is not in the document]
Comment (ksowndar) [reply app/javascript/includes/backtotop.js:2]: It is resolved in [bug/discussion-close](https://github.com/MilesUpHQ/reddit-clone-rails/tree/bug/discussion-close)

File: app/javascript/includes/navbar.js (hunk 2 above)
Comment (ksowndar) [reply app/javascript/includes/navbar.js:2]: It is resolved in [bug/discussion-close](https://github.com/MilesUpHQ/reddit-clone-rails/tree/bug/discussion-close)

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (ksowndar) [reply app/controllers/posts_controller.rb:77]: Yes, we are here updating boolean value of closed to true

**Pull request merged**